infrastructure/saat.tfvars

```diff
@@ -1,2 +1,2 @@
 use_idam_testing_support = "false"
-idam_api_url = "http://idam-api-idam-saat.service.core-compute-saat.internal"
+idam_api_url = "http://betadevbccidamapplb.reform.hmcts.net/"
```

Comment:
Has this been agreed with other teams like CMC and Divorce ?

Reply:
SIDAM have asked all teams to change back to TIDAM / don't use the env